pom.xml

```diff
@@ -1,254 +1,283 @@
+        <excludes>
+          <exclude>beacon/data/*.jsonl</exclude>
+        </excludes>
+      </resource>
+    </resources>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-compiler-plugin</artifactId>
+          <version>3.3</version>
+          <configuration>
+            <source>${target.jdk}</source>
+            <target>${target.jdk}</target>
+            <showWarnings>true</showWarnings>
+          </configuration>
+        </plugin>
+        <plugin>
+          <artifactId>maven-assembly-plugin</artifactId>
+          <version>2.3</version>
+          <configuration>
+            <descriptorRefs>
+              <descriptorRef>jar-with-dependencies</descriptorRef>
+            </descriptorRefs>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.19.1</version>
+          <configuration>
+            <test>*Test</test>
+            <environmentVariables>
+              <generateTestData>${generateTestData}</generateTestData>
+            </environmentVariables>
+            <test>UnitTests</test>
+            <forkCount>3</forkCount>
+            <reuseForks>true</reuseForks>
+            <cleanupDaemonThreads>false</cleanupDaemonThreads>
+            <argLine>-Xmx1536m</argLine>
+          </configuration>
+        </plugin>
+        <plugin>
+          <artifactId>maven-failsafe-plugin</artifactId>
+          <version>3.0.0-M5</version>
+          <executions>
+            <execution>
+              <goals>
+                <goal>integration-test</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.ec4j.maven</groupId>
+          <artifactId>editorconfig-maven-plugin</artifactId>
+          <version>0.1.3						                        						                        						            <!-- check the latest on http://search.maven.org/maven2/org/ec4j/maven/editorconfig-maven-plugin/ --></version>
+          <executions>
+            <execution>
+              <id>check</id>
+              <phase>verify</phase>
+              <goals>
+                <goal>check</goal>
+              </goals>
+            </execution>
+          </executions>
+          <configuration>
+            <!-- See http://ec4j.github.io/editorconfig-maven-plugin/ for full configuration reference -->
+            <excludes>
+              <!-- Note that maven submodule directories and many non-source file patterns are excluded by default -->
+              <!-- see https://github.com/ec4j/editorconfig-linters/blob/master/editorconfig-lint-api/src/main/java/org/ec4j/lint/api/Constants.java#L37 -->
+              <!-- You can exclude further files from processing: -->
+              <exclude>src/main/**/*.whatever</exclude>
```

Comment:
This doesn't match anything. I think you took it from the example and should remove this also.

Reply:
Deleted this